### PR TITLE
added onTimeChange on DatePicker

### PR DESCRIPTION
**Pull request by littletower**
issue #1231

### CODE REVIEW

**Comment by martijnrusschen** [comment]
Please check out the failing build. Also, can you add a test for this fuctionality?

File: package.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "author": "HackerOne",
   "name": "react-datepicker",
   "description": "A simple and reusable datepicker component for React",
-  "version": "1.0.4",
+  "version": "1.1.0",
   "license": "MIT",
-  "homepage": "https://github.com/Hacker0x01/react-datepicker",
+  "homepage": "https://github.com/littletower/react-datepicker",
```
Comment (martijnrusschen) [inline package.json]: Please revert this

File: package.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
   "author": "HackerOne",
   "name": "react-datepicker",
   "description": "A simple and reusable datepicker component for React",
-  "version": "1.0.4",
```
Comment (martijnrusschen) [inline package.json]: Looks like this is based on an older branch?

**Comment by littletower** [comment]
I'll try to take a look at it this week. Thanks

**Comment by martijnrusschen** [comment]
Did you have a chance to look at this? Otherwise, I'm going to clean this from our list.

**Comment by littletower** [comment]
sorry I'm swamped... you can clean it.

**Pull request closed without merging**